chain-impl-mockchain/src/leadership/genesis/mod.rs

```diff
@@ -536,6 +594,33 @@ mod tests {
     use crate::fragment::Contents;
     use crate::header::{BlockVersion, HeaderBuilderNew};
 
+    #[test]
+    pub fn leadership_verify_wrong_proof() {
```

Comment:
could you be more specific on what is the property we're testing?

Reply:
that is most a placeholder name for now as I'm still figuring it out :)
that test is meant to give coverage to line 149, in which the verify method is throwing a verification error